### PR TITLE
new local install

### PR DESCRIPTION
**Pull request by deliciousmonster**
Also, re-order left nav

### CODE REVIEW

File: docs/getting-started/locally.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+NIO [INFO] [main.WebServer] Server 0.0.0.0:8181 started on 0.0.0.0:8181
+NIO [INFO] Component: ServiceManager status changed from: configured to: starting
+NIO [INFO] Component: ServiceManager status changed from: starting to: started
+```
+
+If you see those logs, the nio Platform is up and running. Congratulations!
+
+> By default, new projects use port 8181. If port 8181 isn’t available, you'll see an error. Change the NIOPORT value in your project's nio.env file to fix this error.
+
+> **MacOS Users**: Python 3.6.4+ requires that you install and trust a set of Root Certificates for Python's SSL package. That file is located at /Applications/Python 3.6/Install Certificates.command. Just double-click that file to complete the process. Check /Applications/Python 3.6/ReadMe.rtf for more details.
```
Comment (mattdodge) [inline docs/getting-started/locally.md]: As we've seen, it's not always Python 3.6.4+. I think it more has to do with non-native Python installs. We should clarify and also include an example of the errors that are seen when you need to install the root certs

File: docs/getting-started/locally.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+The cloud is an easy way to get the nio Platform up and running, but to see the power of nio as a distributed system, you should install and run nio on a locally or at the edge.
+
+>**[info] Requirements**
+>
+>* **Python3**: https://www.python.org/downloads (you need Python 3.4+)
+>* **Git**: https://git-scm.com/downloads (nio uses git to manage projects)
+>* **nio**: https://app.n.io/binaries/download (you'll need to agree to the license)
+>  * Windows users should download the Windows Installer. Double-click and follow the prompts.
+
+## Installation
```
Comment (mattdodge) [inline docs/getting-started/locally.md]: Should add a section for downloading the wheel or at least specify that you will need a wheel file. In other words, expand on the "nio" requirement above.

File: docs/getting-started/locally/add-instance.md [hunk 3]
```diff
@@ -1,20 +0,0 @@
...
-1. Complete the **create local instance** window:
-  * In the **instance name** box, enter your instance name.
-  * In the **hostname** box, enter **localhost**.
-  * In the **port** box, enter **8181**.
-  * Leave the **access mode** as **basic**.
-1. Click **accept**.
-
-> Note: When the System Designer connects to a nio instance, it communicates with that instance directly from your browser via an XHR request. Hostnames like `localhost` and other internal IP addresses will work. You must have access to the localhost or other IP address from your machine to use the System Designer.
-
-You may see an issue regarding HTTPS and HTTP instances. Since you launched your instance and presumably didn't load any SSL certificates, the instance is accessible only by HTTP. However, if you are logged into the System Designer via HTTPS, your browser will restrict an XHR request going over HTTP. To connect to a local instance, you can log into the designer via HTTP at http://designer.n.io. All of your instances and systems will be the same, except the nio commands to edit these instances won't happen over HTTPS.
```
Comment (mattdodge) [inline docs/getting-started/locally/add-instance.md]: We should not remove this disclaimer about http/https

File: docs/getting-started/workshops.md [hunk 4]
```diff
@@ -1,12 +1,18 @@
 # Workshops
 
-There’s no “right way” to learn how to use the nio Platform. With nearly unlimited applications and wide-ranging skill sets, everyone can and should take a slightly different approach.  
+If you've gotten this far, you should have a [cloud instance](/getting-started/in-the-cloud.html) AND a [local instance](/getting-started/locally.html) up and running... but what are your instances _doing_? Not a lot, yet... but here's where we change that.
```
Comment (mattdodge) [inline docs/getting-started/workshops.md]: This reads like a guide, docs are not going to be read in order. Instead, this page should describe the workshops and, if necessary, cite the cloud and local instructions as requirements. Is that really a requirement of all workshops though? Or just specific workshops?

File: docs/getting-started/locally.md [hunk 5]
```diff
@@ -5,7 +5,6 @@ The cloud is an easy way to get the nio Platform up and running, but to see the
 >**[info] Requirements**
 >
 >* **Python3**: https://www.python.org/downloads (you need Python 3.4+)
->* **Git**: https://git-scm.com/downloads (nio uses git to manage projects)
```
Comment (mattdodge) [inline docs/getting-started/locally.md]: This is a requirement, right? Why remove it?

File: docs/getting-started/locally.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+Projects are collections of services, blocks and configurations. Create a nio folder to keep things tidy:
+```
+mkdir -p nio/projects && cd nio/projects
+```
+Use the nio CLI (Command Line Interface) to create a new project named "my_project":
+```
+nio new my_project
+```
+**If you'd rather install the Plant Service from our [Distributed Demostration](https://workshops.n.io/plant/virtual/), enter the following:**
+```
```
Comment (kyliedale) [inline docs/getting-started/locally.md]: Do we want this to link them to the `virtual` page or the beginning of the distributed demonstration (the overview of what it is)?

File: docs/getting-started/workshops.md (hunk 4 above)
Comment (kyliedale) [reply docs/getting-started/workshops.md]: All workshops require the cloud. Only the distributed demo (plant workshop) requires local install of nio

File: docs/getting-started/locally.md (hunk 1 above)
Comment (deliciousmonster) [reply docs/getting-started/locally.md]: changed the wording

File: docs/getting-started/locally/add-instance.md (hunk 3 above)
Comment (deliciousmonster) [reply docs/getting-started/locally/add-instance.md]: added back in

File: docs/getting-started/locally.md (hunk 5 above)
Comment (deliciousmonster) [reply docs/getting-started/locally.md]: The pre-installed git on macOS is sufficient.

File: docs/getting-started/locally.md (hunk 6 above)
Comment (deliciousmonster) [reply docs/getting-started/locally.md]: good point... changed

File: docs/getting-started/workshops.md (hunk 4 above)
Comment (deliciousmonster) [reply docs/getting-started/workshops.md]: I removed that first paragraph...

**Pull request merged**